src/main/java/core/basesyntax/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.enums.Color;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstBase;
+    private int secondBase;
+    private int height;
+
+    public IsoscelesTrapezoid(Color color, int firstBase, int secondBase, int height) {
+        super(color);
+        this.firstBase = firstBase;
+        this.secondBase = secondBase;
+        this.height = height;
+    }
+
+    @Override
+    public void draw() {
+        double area = this.calculateArea();
+        String colorLowerCase = this.getColor().name().toLowerCase();
```

Comment:
See `Figure` and `CollorSupllier`. You can store the String in the figure instead of every time calling `name().toLowerCase()`. Applicable for all other figures.